commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -310,6 +372,47 @@ public static CommerceOrder checkoutOrder(CommerceOrder commerceOrder)
 			serviceContext);
 	}
 
+	public static CommerceOrder createCommerceOrderForShipping(
+			long companyId, long userId, long currencyId, long siteGroupId,
+			BigDecimal value)
+		throws Exception {
+
+		CommerceOrder commerceOrder = CommerceTestUtil.addB2CCommerceOrder(
+			companyId, userId, currencyId, siteGroupId);
+
+		int orderStatus = CommerceShipmentConstants.ALLOWED_ORDER_STATUSES[0];
+
+		commerceOrder.setOrderStatus(orderStatus);
```

Comment:
Why grab the first element in `ALLOWED_ORDER_STATUSES`? It seems to me like if we particularly want the first element, we should just use the constant for the first element; if we don't care which element we get, we should select an element from the array randomly -- doing it either of these ways would better document the need we're filling here.